docs/proposals/backup-restore.md

```diff
@@ -260,6 +263,42 @@ registry-b.k8s.io/kube-apiserver@sha256:9081a6f83f4febf47369fc46b6f0f7683c7db243
             └── io.cncf.notary.x509chain.thumbprint#S256: '["xxxxxx"]'
 ```
 
+Back up an image from Docker Hub to local. Upon success, the output in a non-tty mode will be:
+
+```bash
+oras backup docker.io/bitnami/nginx: 1.29.0 --include-referrers --output airgap-snapshot.tar --no-tty
+```
+
+```console
+✔ Found 1 tag in docker.io/bitnami/nginx: 1.29.0
+✔ Starting backup with referrers included...
+
+Pulling content:
+  • application/jose+json                         fe00f3e4bc13    Pulled
+  • application/vnd.cncf.notary.signature         44136fa355b3    Pulled
+  • application/vnd.docker.container.image.v1+json
+    ├─ 6362258f3406                                Pulled
```

Comment:
It MUST show the algorithm like `sha256` and `sha512`.

Reply:
Otherwise, there will be problems with crypto agility.

Reply:
I removed the verbose output changes. ORAS maintainers aligned that we will keep the verbose output as it is now since verbose output it's not for terminal users.